python/glow/gwas/tests/test_lin_reg.py

```diff
@@ -16,13 +17,16 @@ def run_linear_regression(genotype_df, phenotype_df, covariate_df, add_intercept
     Y = phenotype_df.to_numpy('float64', copy=True)
     Y_mask = ~np.isnan(Y)
     Y[~Y_mask] = 0
+    Y -= Y.mean(axis=0)
     Q = np.linalg.qr(C)[0]
-    Y = gwas_fx._residualize_in_place(Y, Q)
+    Y = gwas_fx._residualize_in_place(Y, Q) * Y_mask
+    Y_std_dev = np.sqrt(np.sum(Y**2, axis=0) / (Y_mask.sum(axis=0) - Q.shape[1]))
```

Comment:
It actually might make sense to not do the scaling here since we have no offset. Then we have a simpler baseline to test against. What do you think?